### ISSUE TITLE
Dose anybody know what is a negative cross validation accuracy mean in linear regression model?

### ISSUE THREAD

**Issue by jchen111** [iit-cs579/main#30]
Dose anybody know what is a negative cross validation accuracy mean in linear regression model? We are fitting our data to sklearn linear regression model and get a negative accuracy which really make me confused.
